contracts/contracts/core/BidderRegistry.sol

```diff
@@ -26,10 +26,7 @@ contract BidderRegistry is
      * @dev Modifier to restrict a function to only be callable by the preconfManager contract.
      */
     modifier onlyPreconfManager() {
-        require(
-            msg.sender == preconfManager,
-            "sender is not preconf contract"
-        );
+        require(msg.sender == preconfManager, SenderIsNotPreconfManager());
```

Comment:
I think it would be better if we add some arguments here. For eg. the sender address. The goal here is to have more information, but currently this is not giving any new information.

Reply:
I was trying to get past this PR quickly and minimally introduce custom errors just for gas savings, but you bring up a valid point 👍 I've added args to most errors in this PR